resize.go

```diff
@@ -180,7 +182,18 @@ func transformImage(image *C.VipsImage, o Options, shrink int, residual float64)
 	}
 
 	if o.Force || residual != 0 {
-		image, err = vipsAffine(image, residualx, residualy, o.Interpolator)
+		xshrink := 1.0 / residualx
```

Comment:
Original code used `residualx` and `residualy` here and elsewhere. I followed the same convention for the sake of being consistent and touching as few lines as possible.